crates/re_types_builder/src/codegen/python.rs

```diff
@@ -800,7 +800,7 @@ fn quote_arrow_support_from_obj(arrow_registry: &ArrowRegistry, obj: &Object) ->
 
                 from .{pkg}_ext import {many}Ext # noqa: E402
 
-                class {arrow}(pa.ExtensionType):
+                class {arrow}(pa.ExtensionType): # type: ignore[misc]
```

Comment:
what error are se suppressing here?

Reply:
Some non-sense about inheriting `Any` or something.

I don't remember exactly and didn't bother any further because this is just a verbatim port of what we already do today.